### DISCUSSION TITLE
Hints for better readings on domed face and small openings?

### DISCUSSION THREAD

**Discussion by wenestvedt** [jomjol/AI-on-the-edge-device#3320] [Q&A]
My water meter has a domed glass face, with small openings for each analog digit. (See the attached image.) As a result, many digits are partially obscured and recognition is very unreliable.

Are there any simple changes I can make, like moving the camera farther away or getting a larger camera? Or is this meter simply hopeless for visual recognition? 

Thank you!
![reference](https://github.com/user-attachments/assets/f7f05480-f8d8-4422-bfaa-9e1aec7c807c)

**Comment by SybexX** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-CamZoom

**Reply by wenestvedt** [reply]
Thanks for the link.

Those thirty-five options seem to be very useful for adjusting images where the digits are visible -- but I have images where the digits are obscured or masked. (For example, the 1 and 4 on the left, or the 7 on the right.) Would anything on that page be a better starting point for my problem than the others?

Vielen Dank.

**Comment by SybexX** [comment]
If you zoom in something, it should actually work, otherwise you will have to mount the camera a little further from the meter.

**Reply by wenestvedt** [reply]
Oh! I can do that, I think: I have a case that I can extend. Thank you!
